src/Sylius/Bundle/CoreBundle/Templating/Helper/CurrencyHelper.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Templating\Helper;
+
+use Sylius\Bundle\CurrencyBundle\Templating\Helper\CurrencyHelperInterface;
+use Sylius\Bundle\MoneyBundle\Formatter\MoneyFormatterInterface;
+use Sylius\Component\Currency\Context\CurrencyContextInterface;
+use Sylius\Component\Currency\Converter\CurrencyConverterInterface;
+use Sylius\Component\Currency\Provider\CurrencyProviderInterface;
+use Symfony\Component\Intl\Intl;
+use Sylius\Component\Locale\Context\LocaleContextInterface;
+
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class CurrencyHelper implements CurrencyHelperInterface
```

Comment:
We can go 2 ways here. Extend a helper from _CurrencyContext_ or decorate it. Any way, you should inject all required services and pass it through to _CurrencyHelper_ from _CurrencyContext_. Copy pasting is always a bad solution.